### DISCUSSION TITLE
Welcome to js-good-first-issues-finder Discussions!

### DISCUSSION THREAD

**Discussion by suyash-thakur** [suyash-thakur/js-good-first-issues-finder#1] [Announcements]
## 👋 Welcome!
 We're here to help you become a successful open source developer and contribute to the wider community. Whether you're a beginner or an experienced developer, we're excited to have you join us.

. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by JuanSalvadorMC** [comment]
I'm Salvador, a front-end developer, and I'd like to try to collaborate. Does anyone here know how I could get started? I'd love to contribute.
